source/deployment/on-boarding.rst

```diff
@@ -1,44 +1,73 @@
 Administrator Tasks
 ===================
 
-This document provides instructions for common administrator tasks.
+This document provides instructions for common administrator tasks, including some recommendations on tasks to prepare your Mattermost instance to onboard users.
 
-DO NOT manipulate the Mattermost database
------------------------------------------
+Getting Started Tasks
+-----------------------
+1. Once you've installed and deployed Mattermost, ensure all configuration settings are appropriately set under **System Console > Environment** including: 
+  - Web Server
+  - Database
+  - File Storage
+  - SMTP
+  - Push Notification Server 
+  
+These settings can also be set in the config.json file.  Please see our `configuration settings documenation <../administration/config-settings.rst>`__ for a full listing of all configuration settings. 
 
-  - In particular, DO NOT manually delete data from the database directly. Mattermost is designed as a continuous archive and cannot be supported after manual manipulation.
-  - If you need to permanently delete a team or user, use the `Command Line Tool <../administration/command-line-tools.html>`__.
+2. Adjust settings under **System Console > Site Configuration** to brand and customize how users will interact with the site.  
+Be sure to update the Support Email and Help Link in Mattermost under **System Console > Site Configuration > Customization** to provide your users a resource for password resets or questions on their Mattermost account.
 
-Updating the Support Email & Help Link
----------------------------------------
+ - The Support Email is used on email notifications and during tutorial for users to ask support questions.
+ - The Help Link is on the Mattermost login page, sign-up pages, and Main Menu. Many organizations use this URL to link to their help desk ticketing system.  
+ 
+These settings can also be set in the config.json file.  Please see our `configuration settings documenation <../administration/config-settings.rst>`__ for a full listing of all configuration settings.  
```

Comment:
This and some of the below links on this page redirect to the Github page instead of the documentation.